api/GPUDevice.json

```
@@ -155,6 +155,49 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "bind_GPUTextureView": {
+          "__compat": {
+            "description": "Bind `GPUTextureView` in place of a `GPUExternalTexture`.",
```

The blog post mentions conditions for the bindings. Is this relevant for BCD consumers?

```suggestion
            "description": "Bind `GPUTextureView` (2D, single subresource) in place of a `GPUExternalTexture`.",
```